### ISSUE TITLE
Add support for `accessibilityPosInSet` [Windows Only]

### ISSUE THREAD

**Issue by YajurG** [microsoft/react-native-windows#12511]
Add support for the `accessibilityPosInSet` prop.

## Documentation
[accessibilitPosInSet Documentation](https://microsoft.github.io/react-native-windows/docs/iviewwindowsprops-api)

## Behavior
The `accessibilityPosInSet` prop indicates what position an element is within a set of elements. This prop should take a numeric value.

The value for this prop should be used to set the value of the UIA PositionInSet attached property (See [here](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.automationproperties.positioninset?view=winrt-26100) for documentation on this property). 

## Implementation Plan
We should:
1. Take the value of the `accessibilityPosInSet` prop and set the UIA `PositionInSet` property for that component to be submitted value. 

## To Be Clarified
Note this prop should be specified in tandem with the `accessibilitySetSize` prop. We should consider filing a Yellow or Red Box error if the source only specifies `accessibilityPosInSet`.
